cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,226 @@
+/// <reference types="cypress" />
+
+const years = [
+    2017,
+    2013,
+    2009,
+    2005,
+    2001,
+    1997,
+    1993,
+    1989,
+    1985,
+    1981,
+    1977,
+    1973,
+    1969,
+    1965,
+    1961,
+    1957,
+    1953,
+    1949,
+    1945,
+];
+const yearsWithModernElectionSystem = [2017, 2013, 2009, 2005];
+const yearsWithStLagues = years.slice(0, years.length - 2);
+
+function waitForLoad(iteration: number = 0) {
+    cy.wait(1000).then(() => {
+        cy.get("#page_loader").then((loader) => {
+            if (loader.is(":visible") && iteration < 300) {
+                waitForLoad(iteration++);
+            }
+        });
+    });
+}
+
+describe("ComputationMenu", () => {
+    beforeEach(() => {
+        cy.clearLocalStorage();
+        cy.viewport(1080, 720);
+    });
+
+    context("Default settings", () => {
+        it("Works for all years", () => {
+            cy.visit("").then(() => waitForLoad);
+
+            years.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)